bin/mark-production-changelog.js

```diff
@@ -0,0 +1,87 @@
+#! /bin/node
+
+import fetch from 'node-fetch';
+
+const CHANGELOG_BEARER_TOKEN = process.env.CHANGELOG_BEARER_TOKEN;
+const CHANGELOG_ENDPOINT = 'https://public-api.wordpress.com/wp/v2/sites/wpvipchangelog.wordpress.com/posts';
+
+const REST_END_PAGE_ERROR = 'rest_post_invalid_page_number';
+const RELEASE_CHANNEL_STAGING = 267076;
+const RELEASE_CHANNEL_PRODUCTION = 5905;
+
+const fetchPage = async page => {
+	const headers = {
+		Authorization: `Bearer ${CHANGELOG_BEARER_TOKEN}`,
+	};
+	const queryArgs = {
+		'release-channel': RELEASE_CHANNEL_STAGING,
+		tags: 1784989, // mu-plugins
+		per_page: 100,
+		status: 'draft,publish',
+		page,
+	}
+
+	const argString = Object.keys(queryArgs).map(argument => `${argument}=${queryArgs[argument]}`).join('&');
```

Comment:
```suggestion
	const argString = new URLSearchParams(queryArgs).toString();
```

Reply:
(just in case we might have something that needs to be urlencoded first)

Reply:
Cool.. yeah I was thinking:  I am rediscovering a wheel here right?